app/src/main/java/chat/rocket/android/settings/presentation/SettingsView.kt

```diff
@@ -1,3 +1,9 @@
 package chat.rocket.android.settings.presentation
 
-interface SettingsView
+import chat.rocket.android.core.behaviours.MessageView
+
+interface SettingsView:  MessageView {
+
+    fun showDownloadDialog(error : String?)
```

Comment:
Want to show an error message? You can use the function defined on `MessageView` instead of creating this one.

Reply:
sure